tests/models/test_main.py

```diff
@@ -36,10 +37,30 @@ def test_should_not_return_password_on_user(self):
             password_changed_at=now
         )
 
-        assert user.serialize()['emailAddress'] == "email@digital.gov.uk"
-        assert user.serialize()['name'] == "name"
-        assert user.serialize()['role'] == "buyer"
-        assert 'password' not in user.serialize()
+        serialized_user = user.serialize()
+        assert serialized_user['emailAddress'] == "email@digital.gov.uk"
+        assert serialized_user['name'] == "name"
+        assert serialized_user['role'] == "buyer"
+        assert 'password' not in serialized_user
+
+    def test_should_fall_back_to_created_at_time_if_no_logged_in_at_timestamp(self):
+        self.setup_default_buyer_domain()
+        with freeze_time('2018-01-09'):
+            then = datetime.utcnow()
+            later = then + timedelta(weeks=3)
+            user = User(
+                email_address='email@digital.gov.uk',
+                name='name',
+                role='buyer',
+                password='password',
+                active=True,
+                failed_login_count=0,
+                created_at=then,
+                updated_at=later,
+                password_changed_at=later
+            )
+
+        assert user.serialize()['loggedInAt'] == "2018-01-09T00:00:00.000000Z"
```

Comment:
Very minor quibbling but should this test be explicit and set `logged_in_at=None`? Do we need to set `failed_login_count` if it defaults to 0?

Reply:
Yeah that might make it more obvious exactly what's going on, will update.

Reply:
Fixed.  Turns out we do need to set `failed_login_count` or it complains - it has a not null constraint in the table.